packages/card/PrimaryContent.js

```diff
@@ -24,33 +24,67 @@ import React from 'react';
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 
+import withRipple from '@material/react-ripple';
+
 export default class PrimaryContent extends React.Component {
 
   render() {
     const {
-      className,
-      children,
+      hasRipple,
       ...otherProps
     } = this.props;
-    const classes = classnames('mdc-card__primary-action', className);
-
+    if (hasRipple) {
+      return (<RipplePrimaryContent {...otherProps} />);
+    }
     return (
-      <div
-        className={classes}
-        {...otherProps}
-      >
-        {children}
-      </div>
+      <PrimaryContentDefault {...otherProps} />
     );
   }
 }
 
+
+const PrimaryContentDefault = (props) => {
```

Comment:
Instead of PrimaryContentDefault, can we name it PrimaryContentBase?

Reply:
No problem. That should probably be renamed as well in material-icon:

https://github.com/material-components/material-components-web-react/blob/master/packages/material-icon/index.js#L57-L80